apps/docs/self-hosting/docker-compose.mdx

```diff
@@ -98,7 +98,13 @@ This command will:
 
 ## Using a Reverse Proxy
 
-By default the app will run unencrypted on port 3000. If you want to serve the app over HTTPS you will need to use a reverse proxy.
+By default the app will run unencrypted on port 3000. If you want to serve the app over HTTPS you will need to use a reverse proxy. For example, using Apache you would put something similar to the following in your web site's configuration:
+
+```
+ ProxyPreserveHost On
+ ProxyPass / http://localhost:3000/
+ ProxyPassReverse / http://localhost:3000/
+```
```

Comment:
That's rather too much information :-) My intention was simply to orientate a moderately experienced user in what was involved (and they may in any case be running nginx). This would be similar to the fact that reasonable knowledge of Docker is also assumed.

Reply:
OK, although I think "To secure the application via HTTPS" may be misleading because that's dependent on other things.

Reply:
Again, I'm not trying to describe in detail how to set up a reverse proxy. That's not in scope, and if it was you'd need to do the same for Docker. The suggestions I made are fine in my opinion.

I don't want to confuse things, so maybe just leave it as is and close this PR.

Reply:
It would be nice to know what the maintainer(s) of this repo think.